src/components/dev-grid-utils.js

```diff
@@ -1,67 +1,64 @@
-    photo:
-      'https://www.placement.iitbhu.ac.in/media/user_dp/2019/met/yash.ajitpatil.met19.png',
-    siteUrl: null,
-  },
-  {
-    githubUrl: "https://github.com/agungjsp",
-    githubUsername: "agungjsp",
-    linkedinUrl: null,
-    photo: "https://avatars.githubusercontent.com/u/30463526?v=4",
-    siteUrl: null,
-  },
-  {
-    githubUrl: 'https://github.com/',
-    githubUsername: null,
-    linkedinUrl: 'https://www.linkedin.com/',
-    photo: null,
-    siteUrl: null,
-  },
-  {
-    githubUrl: 'https://github.com/',
-    githubUsername: null,
-    linkedinUrl: 'https://www.linkedin.com/',
-    photo: null,
-    siteUrl: null,
-  },
-];
+export const profiles = [{
+        githubUrl: 'https://github.com/MightyJoeW',
+        githubUsername: 'mightyjoew',
+        linkedinUrl: 'https://www.linkedin.com/in/josephmwarren/',
+        photo: 'https://avatars.githubusercontent.com/u/24509848?v=4',
+        siteUrl: 'https://www.getscriptordietryin.com/',
+    },
+    {
+        githubUrl: 'https://github.com/MrsWarren',
+        githubUsername: 'MrsWarren',
+        linkedinUrl: 'https://www.linkedin.com/in/krystleawarren/',
+        photo: 'https://avatars.githubusercontent.com/u/31019414?v=4',
+        siteUrl: null,
+    },
+    {
+        githubUrl: 'https://github.com/mariothedog',
+        githubUsername: 'Mariothedog',
+        linkedinUrl: null,
+        photo: 'https://avatars.githubusercontent.com/u/28541089?v=4',
+        siteUrl: null,
+    },
+    {
+        githubUrl: 'https://github.com/nlewis84',
+        githubUsername: 'nlewis84',
+        linkedinUrl: 'https://www.linkedin.com/in/nlewis84',
+        photo: 'https://media-exp1.licdn.com/dms/image/C4E03AQF6GRty7hLmdA/profile-displayphoto-shrink_400_400/0/1619664151128?e=1639008000&v=beta&t=E5bVRZmjH3EH2fAKBdA3YQ-FHBYb_WHGdYv5tNM6iRo',
+        siteUrl: 'https://nathanlewis.dev',
+    },
+    {
+        githubUrl: 'https://github.com/Wijayaac',
+        githubUsername: 'Wijayaac',
+        linkedinUrl: 'https://www.linkedin.com/in/kadek-wijaya-217577212/',
+        photo: 'https://avatars.githubusercontent.com/u/60057324?v=4',
+        siteUrl: 'https://wijayaac.netlify.app/',
+    },
+    {
+        githubUrl: 'https://github.com/Yash-271120',
+        githubUsername: 'Yash-271120',
+        linkedinUrl: 'https://www.linkedin.com/in/yash-patil-3a163519b/',
+        photo: 'https://www.placement.iitbhu.ac.in/media/user_dp/2019/met/yash.ajitpatil.met19.png',
+        siteUrl: null,
+    },
+    {
+        githubUrl: "https://github.com/agungjsp",
+        githubUsername: "agungjsp",
+        linkedinUrl: null,
+        photo: "https://avatars.githubusercontent.com/u/30463526?v=4",
+        siteUrl: null,
+    },
+    {
+        githubUrl: "https://github.com/rohitkothapalli",
+        githubUsername: "rohitkothapalli",
+        linkedinUrl: "https://www.linkedin.com/in/rohit-kothapalli-079719194/",
+        photo: null,
```

Comment:
@rohitkothapalli Did you want to include your profile pic?

Reply:
No sir .